ref_ts/templates/pages/reftimeseriesresource.html

```diff
@@ -1,114 +1,107 @@
-                        <hr style="margin-top: 0px;margin-bottom: 2px">
-                        {% if variable.code %}
+            {% endif %}
+
+            {% if variable and variable.code or variable and variable.name %}
+                <legend>Variable</legend>
+                <div class="space-bottom">
+                    {% if variable.code %}
                         <div class="row">
                             <div class="col-sm-4"><strong>Code:</strong></div>
                             <div class="col-sm-8">{{ variable.code }}</div>
                         </div>
-                        {% endif %}
-                        {% if variable.name %}
+                    {% endif %}
+                    {% if variable.name %}
                         <div class="row">
                             <div class="col-sm-4"><strong>Name:</strong></div>
                             <div class="col-sm-8">{{ variable.name }}</div>
                         </div>
-                        {% endif %}
                     {% endif %}
-                    <hr style="border:0">
                 </div>
-                <div>
-                    {% if method and method.code or method and method.description %}
-                        <h4><strong>Method:</strong></h4>
-                        <hr style="margin-top: 0px;margin-bottom: 2px">
-                        {% if method.code  %}
+            {% endif %}
+
+            {% if method and method.code or method and method.description %}
+                <legend>Method</legend>
+                <div class="space-bottom">
+                    {% if method.code %}
                         <div class="row">
                             <div class="col-sm-4"><strong>Code:</strong></div>
                             <div class="col-sm-8">{{ method.code }}</div>
                         </div>
-                        {% endif %}
-                        {% if method.description  %}
+                    {% endif %}
+                    {% if method.description %}
                         <div class="row">
                             <div class="col-sm-4"><strong>Description:</strong></div>
                             <div class="col-sm-8">{{ method.description }}</div>
                         </div>
-                        {% endif %}
-                    <hr style="border:0">
                     {% endif %}
                 </div>
-                <div>
-                    {% if quality_level and quality_level.code and quality_level and quality_level.definition %}
-                        <h4><strong>Quality Control Level:</strong></h4>
-                        <hr style="margin-top: 0px;margin-bottom: 2px">
-                        {% if quality_level.code %}
+            {% endif %}
+
+            {% if quality_level and quality_level.code or quality_level and quality_level.definition %}
+                <legend>Quality Control Level</legend>
```

Comment:
Good choice using <legend> for a subtitle.

Reply:
The subtitle "Extent". Should it also be changed to <legend> for consistency purpose?

Reply:
We could, I'll try to make this change after the release. For now we'll leave it like this since the only difference is a bit of spacing.